other-components-docker.yml

```diff
@@ -0,0 +1,6 @@
+# This file lists software components, and their versions, that are part of a
+# Mender release, have Docker containers, but are not part a default Mender
+# installation (have no docker-compose file).
+services:
+    mtls-ambasador:
```

Comment:
`s/ambasador/ambassador/`

Reply:
Thanks. I'll check for more occurrences of this typo.